CLI/support/sparkle_job_help.py

```diff
@@ -125,17 +128,75 @@ def get_running_jobs() -> list[SlurmRun]:
             if run.status == Status.WAITING or run.status == Status.RUNNING]
 
 
+def get_dependencies(jobs: list[SlurmRun]) -> list[SlurmRun]:
+    """Adds Dependencies. Should be removed once RunRunner provides this feature."""
+    for job in jobs:
+        with job.json_filepath.open() as file:
+            data = json.load(file)
+        if len(data["dependencies"]) > 0:
+            job_ids = [dep["run_id"] for dep in data["dependencies"]]
+        else:
+            job_ids = []
+        job.dependencies = job_ids
+    return jobs
+
+
+def clear_console(lines: int) -> None:
```

Comment:
Generalise more, clear_console_lines(num_lines: int), update comment (Don't describe the table)